packages/react/src/components/ComboBox/ComboBox-story.js

```diff
@@ -70,6 +70,9 @@ export const combobox = () => (
       placeholder="Filter..."
       titleText="ComboBox title"
       helperText="Combobox helper text"
+      ref={() => {
+        console.log('some ref');
+      }}
```

Comment:
```suggestion
```

I think we should remove this bit from the story, dunno how helpful it is and can add noise when developing or debugging.